array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,19 @@
+/* exported includes */
+function includes(array, value) {
+  if (array.length === 0) {
+    return false;
+  }
```

Comment:
This isn't necessary. If the array length is zero, the loop will never start, and you'll end up with the same situation of returning `false` at the end of the function